### ISSUE TITLE
Goals  stop scheduling  [BUG]

### ISSUE THREAD

**Issue by Tarawssmith** [rosenpin/Goaly#19]
I used the app for two weeks or so and then it just stopped scheduling goals.

 - Device: Pixel 4
 - OS: Android 13
 - App Version 1.0.1

**Comment by Frayo44** [comment]
Goaly schedules goals each time for the 2 upcoming weeks. Do you have place for your goals in the calendar in the upcoming weeks?

Sometimes all-day events make it seem like there is no space for events and it looks like Goaly does not schedule events for the next weeks because it doesn't find time for them.

**Comment by Tarawssmith** [comment]
I have plenty of time and no all day events ..

Musically
Tara Smith
________________________________

**Comment by rosenpin** [comment]
Can you send a screenshot of the main screen of Goaly?  
Also, make sure that the Goaly calendar is active in your Google Calendar like so:  
<img width="234" alt="image" src="https://user-images.githubusercontent.com/5289630/208291348-96457331-792b-4254-a318-e694f74f5970.png">

**Comment by Tarawssmith** [comment]
Here you go

Musically

Tara Smith

________________________________

**Comment by rosenpin** [comment]
I can't see the attached image, please upload it to github using this link and not via email: https://github.com/rosenpin/Goaly/issues/19

**Comment by Tarawssmith** [comment]
![Screenshot_20221218-075244](https://user-images.githubusercontent.com/120754122/208323227-ab850362-cf8e-41ff-b54e-32d2012d882a.png)
![Screenshot_20221218-075229](https://user-images.githubusercontent.com/120754122/208323234-ab68598b-4a40-4ff4-8d3c-3e854b3d704e.png)

**Comment by rosenpin** [comment]
Thank you!  
Can you please try to uninstall then reinstall the app and let us know if it fixes the issue?  
When reinstalling - make sure you sign in with the same Google account you used previously

**Comment by Tarawssmith** [comment]
Tried that, no change

**Comment by Tarawssmith** [comment]
Now what?

**Comment by rosenpin** [comment]
Do you have goals scheduled for the coming week/next week?  

We're working on way to test this bug further, we will send you an APK to test along with instructions when done

**Comment by Tarawssmith** [comment]
Actually I finally got some scheduled starting Dec 31st

Musically

Tara Smith

________________________________

**Comment by rosenpin** [comment]
I see  
Goaly only schedules 2 weeks in advance on a weekly basis. This is done so Goaly will only schedule your goals once your schedule is ready   
If you didn't see goals scheduled for 2 weeks in advance, that's normal  
This behavior might not be clear enough, we will add this to the instructions

**Comment by Tarawssmith** [comment]
Ok, I get it.  I wish it would fill in more often so when an event gets canceled that time can fill in with goals.  Also it would be great if new events would move goals and reschedule them. Thanks for your help.

Musically,
Tara Smith

________________________________

**Comment by rosenpin** [comment]
Yes, that will be a great feature! I will open a new issue for this.  
Thank you for your support
